volatility3/framework/plugins/windows/prefetch.py

```diff
@@ -0,0 +1,535 @@
+# References :
```

Comment:
This should really be a separate PR please...  5:S  I've let this kind of stuff slide, but we *really* need to get in the habit of separating these things out, even if one depends on the other.  This PR is about fixing up the Registry Exceptions, not about adding the prefetch plugin...

Reply:
that's my bad, it's something I had started on a while ago and just had the changed code but added it by accident.  it shouldn't be here as it probably doesn't even work.

Reply:
All good, it was just github hides it as being a big change by default so easy to miss.  Pull it out and the rest is pretty much good to go once the registry exceptions have been combined.  5:)